opmon-protobuf-dbwriter/opmon-dbwriter-deployment.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    kluctl.io/skip-delete-if-tags: "true"
+  labels:
+    pod-security.kubernetes.io/audit: baseline
+    pod-security.kubernetes.io/audit-version: latest
+    pod-security.kubernetes.io/enforce: baseline # unified image runs as root :(
+    pod-security.kubernetes.io/enforce-version: latest
+    pod-security.kubernetes.io/warn: baseline
+    pod-security.kubernetes.io/warn-version: latest
+  name: ers
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/app: opmon-protobuf-dbwriter
+    app.kubernetes.io/component: opmon-protobuf-dbwriter
+  name: opmonstream-dbwriter
+  namespace: opmon
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: opmon-protobuf-replica
+  template:
+    metadata:
+      labels:
+        app: opmon-protobuf-replica
+        app.kubernetes.io/app: opmon-protobuf-dbwriter 
+        app.kubernetes.io/component: opmon-protobuf-dbwriter  
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: node-role.kubernetes.io/worker
+                operator: Exists
+      containers:
+      - image: ghcr.io/dune-daq/microservices:mroda-opmon_protobuf   ## TBC
+        imagePullPolicy: Always
+        name: opmon-protobuf-replica 
+        env:
```

Comment:
Should this be updated to include the user/pass values?

Reply:
But this is not there yet, right? I added the option in the python, but the cluster still don't have that implemented. Until then there is not much we can do. Unless you want to change this while we add the microservice.